docs/application/native/guides/multimedia/media-vision.md

```diff
@@ -7,15 +7,23 @@ You can use the following visual detection and recognition features in your nati
 - [Barcode Detection and Generation](image-barcode.md)
   
   You can perceive and understand an image or extract information from images in your application. You can encrypt a message by generating a barcode from it, and you can also detect barcodes in an image or from a camera preview stream.
+
 - [Face Detection, Recognition, and Tracking](face-detection.md)
-  
+
   You can detect a face from an image, and recognize a face with a set of examples. You can also track a face within the camera preview stream.
+
+- [Deep Learning Based Face Recognition](face-recognition.md)
+
+  You can register, unregister and recognize a face in your application with more accuracy, which uses Deep Learning technique.
```

Comment:
@daeinki Also I think these headings should be placed in alphabetical order. So Deep Learning Based Face Recognition should be placed before Face Detection, Recognition, and Tracking. Please address this

Reply:
Got it. Thanks.